api/src/main/java/me/lokka30/treasury/api/economy/account/SharedAccountPermission.java

```diff
@@ -11,42 +11,42 @@
 import me.lokka30.treasury.api.economy.transaction.EconomyTransactionInitiator;
 
 /**
- * Enum that holds the permissions of a {@link BankAccount}.
+ * Enum that holds the permissions of an {@link Account} that is shared among multiple players.
  *
  * @author MrNemo64
- * @see BankAccount
+ * @see Account
  * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
  */
-public enum BankAccountPermission {
+public enum SharedAccountPermission {
```

Comment:
Maybe we can just name it AccountPermission? I'm fine with that name too but what about AccountPermission?